.github/ISSUE_TEMPLATE/feature.yaml

```diff
@@ -0,0 +1,58 @@
+name: 🚀 Feature
+description: Propose a new feature or enhancement to the Prometheus operator
+labels: ["kind/feature", "needs-triage"]
```

Comment:
Does it make sense "needs-triage" in this context?

Reply:
Ya I thought that would help us to know Promethues Operator team accepted feature request or not. I have seen sometimes contributors pick features un-triaged ones and sometimes that doesn't get merged. So this might help to reduce that 🙂